Detectors/ITSMFT/MFT/simulation/src/MFTSimulationLinkDef.h

```diff
@@ -18,4 +18,6 @@
 #pragma link C++ class o2::base::DetImpl < o2::mft::Detector> + ;
 #pragma link C++ class o2::mft::DigitizerTask + ;
 
+#pragma link C++ function o2::mft::test::addAlignableVolumes;
```

Comment:
what is this test namespace? `addAlignableVolumes` is a method of `Detector` class.

Reply:
yes indeed, it was used later for a class test but useless here.